spring-restdocs-asciidoctor-extensions/build.gradle

```diff
@@ -0,0 +1,15 @@
+description = 'Spring REST Docs Asciidoctor Extensions'
+
+dependencies {
+	compile 'org.asciidoctor:asciidoctorj:1.5.2'
+	optional 'junit:junit'
+}
+
+task copyTestAsciidoctorFile(type: Copy) {
```

Comment:
If this copy is necessary, I'd rather do it as part of the test than in the Gradle build so that it happens when running things in an IDE too.